### DISCUSSION TITLE
Docs with Mermaid Diagram for team consumption

### DISCUSSION THREAD

**Discussion by marlonjames71** [mermaid-js/mermaid#5380] [Q&A]
I got tasked at work with figuring out a great way to create documentation that has diagrams showing how all of our tech fits together.

I think Mermaid could be just that. However, there are some main things I'm wondering about as well as some constraints and I would like to see if this community has some good ideas or things to look further into:

- How are other teams doing this?
- Is there a nice way that other team members can view these docs and see the diagrams rendered? If so, which software (if any) is required for this?
- Would like the work on these docs to be tracked via git
- Would also like the docs to live in its own directory in our code base for easy accessibility (open to other ideas)

So far, Gitbook looks to be a very nice solution, but I don't see any syncing with Bitbucket (what we use) and I'm trying to steer clear of needing another service. We're in the Atlassian ecosystem.

Any help would be greatly appreciated. Thank you!

**Comment by samelawrence** [comment]
We keep our Mermaid diagrams in Notion, where it has 1st-party support as a native language selection option in `Code` blocks. We don't track our Mermaid code in git - and I think if you wanted to do this you'd be doing a lot of copy/pasting back and forth, so it might not be a good solution for you. Notion is where we track all our documentation, so it makes sense for everything to live there, for our team.

**Reply by marlonjames71** [reply]
Thanks for replying. I thought of that as well. Only thing is being a big company, it would be very hard to convince them to switch from Atlassian/Confluence to Notion. I did find that Gitbook could be a good option. But that would take some convincing as well.

**Reply by d48** [reply]
Fairly low cost if this works

https://marketplace.atlassian.com/apps/1222572/mermaid-charts-diagrams-for-confluence

**Reply by marlonjames71** [reply]
I ended up seeing that too. Thanks!
